packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -132,6 +132,103 @@ void main() {
     expect(material.elevation, elevation);
     expect(material.shape, shape);
   });
+
+  testWidgets('BottomSheet modalElevation takes priority over elevation - Modal', (WidgetTester tester) async {
+    const double modalElevation = 5.0;
+    const double persistentElevation = 7.0;
+    const BottomSheetThemeData bottomSheetTheme = BottomSheetThemeData(
+      elevation: persistentElevation,
+      modalElevation: modalElevation,
+    );
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
```

Comment:
consider inserting new line before this line. The `await tester.pumpWidget` is the first "action"

http://wiki.c2.com/?ArrangeActAssert

Reply:
Done